src/mmw/js/src/modeling/views.js

```diff
@@ -874,13 +874,15 @@ var ScenarioToolbarView = Marionette.CompositeView.extend({
         var gisData = this.currentConditions.getGisData().model_input,
             isGwlfe = this.modelPackage === coreUtils.GWLFE && !_.isEmpty(gisData),
             isOnlyCurrentConditions = this.collection.length === 1 &&
-                this.collection.first().get('is_current_conditions');
+                this.collection.first().get('is_current_conditions'),
+            createdByCurrentUser = this.model.get('user').id === App.user.get('id');
```

Comment:
Prefer reusing the [`isEditable`](https://github.com/WikiWatershed/model-my-watershed/blob/986f51fa7d46eb2ece2a53a8776c5cfb75ca4f77/src/mmw/js/src/modeling/views.js#L1144-L1146) method here, as it is elsewhere in this file.